src/components/Disputes/panels/AppealPanel.js

```diff
@@ -215,19 +215,17 @@ const AppealPanel = React.memo(function AppealPanel({
   )
 })
 
-export default ({ dispute, confirm, ...props }) => {
+function Panel({ dispute, confirm, ...props }) {
   const { appeal } = getDisputeLastRound(dispute)
-  // Cases where a confirm appeal is done, the next round is created (with no appeal) and the panel hasn't closed yet
-  if (confirm && !appeal) {
-    return null
-  }
 
-  return (
-    <AppealPanel
-      dispute={dispute}
-      appeal={appeal}
-      confirm={confirm}
-      {...props}
-    />
-  )
+  return useMemo(() => {
```

Comment:
Ah interesting, I meant `React.memo()` not `useMemo()` previously, but I've never considered using a `useMemo()` for the rendered component before.

Although in this case, I think it'd be functionally equivalent to use `React.memo()` (and it'd be a bit cleaner).